packages/contentstack-export-to-csv/src/util/config.js

```diff
@@ -1,10 +1,11 @@
 module.exports = {
   cancelString: 'Cancel and Exit',
   exportEntries: 'Export entries to a .CSV file',
-  exportUsers: "Export organization users' data to a .CSV file",
+  exportUsers: "Export organization user's data to a .CSV file",
+  exportTeams: "Export organization team's data to a .csv file",
   adminError: "Unable to export data. Make sure you're an admin or owner of this organization",
   organizationNameRegex: /\'/,
   CLI_EXPORT_CSV_LOGIN_FAILED: "You need to login to execute this command. See: auth:login --help",
-  CLI_EXPORT_CSV_ENTRIES_ERROR: "You need to either login or provide a management token to execute this command"
-
+  CLI_EXPORT_CSV_ENTRIES_ERROR: "You need to either login or provide a management token to execute this command",
+  CLI_EXPORT_CSV_API_FAILED: 'Something went wrong. Please try again!'
```

Comment:
Is this text reviewed by doc team?

Reply:
Yes this one is reviewed it is taken from taxonomy